generate-docs/scripts/preprocessor.ts

```diff
@@ -10,60 +10,56 @@ tryCatch(async () => {
     // Display prompts
     // ----
     console.log('\n\n');
-    const urlToCopyOfficeJsFrom = await promptFromList({
-        message: `What is the source of the Office-js TypeScript definition file that should be used to generate the RELEASE docs?`,
-        choices: [
-            { name: "DefinitelyTyped", value: "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js/index.d.ts" },
-            { name: "Prod CDN", value: "https://appsforoffice.officeapps.live.com/lib/1.1/hosted/office.d.ts" },
-            { name: "Local file [generate-docs\\script-inputs\\office.d.ts]", value: "" }
-        ]
-
-        // Note: using "appsforoffice.officeapps.live.com" instead of "appsforoffice.microsoft.com"
-        //     to avoid being redirected to the EDOG environment on corpnet.
-        // If we ever want to generate not just public d.ts but also "office-with-first-party.d.ts",
-        //     replace the filename.
-    });
 
     console.log('\n');
-    const urlToCopyPreviewOfficeJsFrom = await promptFromList({
-        message: `What is the source of the Office-js TypeScript definition file that should be used to generate the PREVIEW docs?`,
+    const sourceChoice = await promptFromList({
+        message: `What is the source of the Office-js TypeScript definition files that should be used to generate the docs?`,
```

Comment:
Do we want to note that fallback is DT (like for CDN)?

Reply:
The source gets logged later. That seems good enough to debug any potential issues.